zerodb/models/__init__.py

```diff
@@ -76,7 +76,7 @@ def __setattr__(self, name, value):
         origattr = getattr(self, name, None)
         if origattr is not None and name in self._z_indexed_fields and \
                 not isinstance(origattr, fields.Indexable):  # reindex notify
-            modified(self)
+            modified(self, name)
```

Comment:
This wouldn't raise a ModelException if the document wasn't indexed to start with? E.g. if we do

```
obj = Something(a=1, b=2)
obj.a = 3  # <- should it make a call to reindex for the field a even if the object doesn't have uids and not indexed?
db.add(obj)
transaction.commit()
```

Even if it is all fine, I'd add something like this to the tests to avoid regressions